openwisp_monitoring/monitoring/static/monitoring/js/chart.js

```diff
@@ -57,10 +72,6 @@
         if (type === 'histogram') {
             layout.hovermode = 'closest';
         }
-        if (type === 'stackedbar') {
-            type = 'bar';
-            layout.barmode = 'stack';
-        }
```

Comment:
why is this being removed?

Reply:
this is being handled here 
https://github.com/openwisp/openwisp-monitoring/blob/8536dd63a8e4db1c6df47d0a553706218a4e3fe3/openwisp_monitoring/monitoring/static/monitoring/js/chart.js#L123-L134